pages/index.rst

```diff
@@ -22,14 +22,17 @@ What are the advantages of HPy?
 
 - **Much faster** on alternative implementations such as PyPy_, GraalPython_.
 
-- **Debug Mode**: in debug mode, you can easily identify common problems such
-  as memory leaks, invalid lifetime of objects, invalid usage of APIs. Have
-  you ever forgot a ``Py_INCREF`` or ``Py_DECREF``? The HPy debug mode will
-  detect these mistakes for you.
-
 - **Universal binaries**: extensions built for the *HPy Universal ABI* can be
   loaded unmodified on CPython, PyPy, GraalPython, etc.
 
+- **A migration path** for mixing legacy C-API calls with HPy API calls. Once
+  all the code is migrated, the extension can be compiled as a universal binary that works on any CPython version, PyPy, or GraalPy.
+
+- `Debug mode`_: in debug mode, you can easily identify common problems such
```

Comment:
```suggestion
- **Debug mode**: in debug mode, you can easily identify common problems such
```
The formatting should be bold here like the others, yes?

Reply:
I think making it a link is more important, and didn't know how to do both bold and a link.

Reply:
Oh, I see. Maybe we could just add the link next to it in the start of the text? Maybe on the "debug mode" in "in debug mode"?

Reply:
Good idea, done.